### DISCUSSION TITLE
Quasar 2.12 q-drawer transparency

### DISCUSSION THREAD

**Discussion by codingwarriortech** [quasarframework/quasar#15904] [General - Components / Directives / etc] [answered]
Hi,

does someone know ho to get a transparent q-drawer with Quasar v2 ?

Thanks a lot.

**Comment by suleimanas** [comment, marked as answer]
you can always use css :)
in the main app css files I always make the drawer's background transparent then if I want I can set a bg color class otherwise it'll be transparent all the time

```
.q-drawer {
  background: transparent
}
```

**Reply by codingwarriortech** [reply]
Hi suleimanas,

 I've had attached an own class to the q-drawer and set background-color: rgba(0, 0, 0, 0.5) for it, but the background was not visible. You're right, using it directly on the .q-drawer class makes it work. As I have only one q-drawer, this fixes it for me.

Thanks a lot -)!
